kiwipy/rmq/communicator.py

```diff
@@ -262,69 +263,81 @@ async def _send_response(self, reply_to, correlation_id, response):
 
 class RmqCommunicator:
     """
-    A publisher and subscriber using aio_pika and an asyncio event loop
+    A publisher and subscriber using aio_pika and an asyncio event loop.
+
+    The communicator takes an :class:`aio_pika.Connection` as there can be multiple communicators
+    that share a single connection.
```

Comment:
Is this correct? I only see `connection_params` and `connection_factory` in constructor args, none of which the type seems to be `aio_pika.Connection`.

Reply:
Thanks!  This _was_ the case, but no longer.  I'll push a correction.

Reply:
Ok, done.